css/contact-us.css

```diff
@@ -188,9 +253,6 @@ textarea.form-control__input {
   background-color: #0D0D15;
```

Comment:
@Alinchik1 you mixed different classes for .form-checkbox in CSS.
Now it's:
`.form-radio__box::after {...}
.form-control__input:checked + .form-checkbox__inner .form-checkbox__box::after {}`

Reply:
And don't forget to set different background for ::after element in :checked state, because now they are the same:
246 line (background-color: #0D0D15;) and 253 line (background-color: #0D0D15;)

Reply:
Here I did a lot of mess, we were styling those checkboxes during the lesson and I put everything straight to my CSS but later on I changed the HTML code so the classes don't fit.
I will try to fix it.

Reply:
Fixed some not sure if all but style looks ok